roles/installer/templates/deployments/deployment.yaml.j2

```diff
@@ -361,6 +353,75 @@ spec:
 {% if ee_extra_volume_mounts -%}
             {{ ee_extra_volume_mounts | indent(width=12, first=True) }}
 {% endif %}
+        - image: '{{ _image }}'
+        name: '{{ ansible_operator_meta.name }}-rsyslog'
+        imagePullPolicy: '{{ image_pull_policy }}'
+        {% if task_privileged == true %}
+        securityContext:
+          privileged: true
+        {% endif %}
+        {% if task_command %}
+        command: {{ task_command }}
+        {% endif %}
+        {% if task_args %}
+        args: {{ task_args }}
+        {% endif %}
+        volumeMounts:
+        {% if bundle_ca_crt %}
+          - name: "ca-trust-extracted"
+            mountPath: "/etc/pki/ca-trust/extracted"
+          - name: "{{ ansible_operator_meta.name }}-bundle-cacert"
+            mountPath: /etc/pki/ca-trust/source/anchors/bundle-ca.crt
+            subPath: bundle-ca.crt
+            readOnly: true
+        {% endif %}
+          - name: "{{ ansible_operator_meta.name }}-application-credentials"
+            mountPath: "/etc/tower/conf.d/execution_environments.py"
+            subPath: execution_environments.py
+            readOnly: true
+          - name: "{{ ansible_operator_meta.name }}-application-credentials"
+            mountPath: "/etc/tower/conf.d/credentials.py"
+            subPath: credentials.py
+            readOnly: true
```

Comment:
this is needed because `rsyslog-configurer` need to be able to communicate with the database